Binary/Loader/GridFSLoader.php

```diff
@@ -1,54 +0,0 @@
-<?php
-
-/*
- * This file is part of the `liip/LiipImagineBundle` project.
- *
- * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
- *
- * For the full copyright and license information, please view the LICENSE.md
- * file that was distributed with this source code.
- */
-
-namespace Liip\ImagineBundle\Binary\Loader;
-
-use Doctrine\ODM\MongoDB\DocumentManager;
-use Liip\ImagineBundle\Exception\Binary\Loader\NotLoadableException;
-
-class GridFSLoader implements LoaderInterface
```

Comment:
Flysystem also has support for GridFS (https://flysystem.thephpleague.com/adapter/gridfs/), so people who rely on GridFS could switch to that as well if need be.